### ISSUE TITLE
gcc 6.1 warning in libzfs_import.c

### ISSUE THREAD

**Issue by Bronek** [zfsonlinux/zfs#4674]
When building zfs-utils under gcc 6.1 I noticed following warning
```
libzfs_import.c: In function 'fix_paths':
libzfs_import.c:224:28: warning: self-comparison always evaluates to true [-Wtautological-compare]
    if (best->ne_num_labels == best->ne_num_labels &&
                            ^~
```

**Comment by DeHackEd** [comment]
Already fixed in 20c901dc7a96a5f156d28bfb26d9f82026a4fcfb

**Comment by Bronek** [comment]
duplicate of #4632 , thanks @DeHackEd !
